tools/ConvertTo-PseudoLocalization.ps1

```diff
@@ -60,19 +60,16 @@ $content.PreserveWhitespace = $true
 $content.Load($Path)
 
 function GetPseudoLocalization([string]$key, [string]$value, [string]$comment) {
-    $locked = $null
-    if ($comment -match '.*\{Locked=?([^}]*)\}.*') {
-        $locked = $Matches[1]
-    }
+    $placeholders = @{}
 
-    # Skip {Locked} and {Locked=qps-ploc} entries
-    if ($locked -and (($locked -eq '') -or $locked.Contains('qps-ploc'))) {
-        continue
-    }
+    if ($comment -match '.*\{Locked=?([^}]*)\}.*') {

```

Comment:
Unless you are trying to capture the explicit text *around* `{Locked...}`, you can drop the `.*` at the beginning and the of this regex.

Reply:
Weird, when I previously used `-replace` it didn't work properly if I dropped the `.*`. Let me try...